### PR TITLE
Adding gplately and platetectonictools

### PR DESCRIPTION
**Pull request by jcannon-gplates**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by jcannon-gplates** [comment]
@conda-forge-admin please rerender

**Comment by jcannon-gplates** [comment]
@conda-forge/help-python, ready for review!

File: recipes/gplately/LICENSE
Comment (dopplershift) [inline recipes/gplately/LICENSE:1]: Where did this license come from? I don't see it anywhere upstream, and if they don't have an explicit license, we can't distribute it.

Comment (brmather) [reply recipes/gplately/LICENSE:1]: Hi @dopplershift. My apologies, the license has now been included upstream:
https://github.com/GPlates/gplately/blob/master/LICENSE

**Comment by brmather** [comment]
By the way, I am happy to be a maintainer with @jcannon-gplates

File: recipes/gplately/LICENSE
Comment (dopplershift) [reply recipes/gplately/LICENSE:1]: :+1:

File: recipes/platetectonictools/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0]|lower }}/{{ name|lower }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 3935e7846f7c16040a04ca689b10be8378afe6573dc8c4eaae0cf6b75f33c07a
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
```
Comment (dopplershift) [inline recipes/platetectonictools/meta.yaml:14]: This looks like it's pure Python and can be noarch:
```suggestion
build:
  number: 0
  script: "{{ PYTHON }} -m pip install . -vv"
  noarch: python
```

File: recipes/platetectonictools/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    - pygplates
+    - scipy
+    - cartopy
+  run:
+    - setuptools
+    - numpy
+    - python
+    - pygplates
+    - scipy
+    - cartopy
```
Comment (dopplershift) [inline recipes/platetectonictools/meta.yaml:30]: For noarch. Also, most of the stuff in host doesn't seem to be needed at package build time:
```suggestion
  host:
    - pip
    - python >=3.5
  run:
    - setuptools
    - numpy
    - python >=3.5
    - pygplates
    - scipy
    - cartopy
```

**Review by dopplershift — CHANGES_REQUESTED** [review]
Just a few minor tweaks to make `platetectonictools` a noarch package and this is good to go.

**Comment by jcannon-gplates** [comment]
Thanks for the review. I made the suggested changes. However if now fails to build on osx and win (but succeeds on linux). Previously it succeeded on all three.

I'm not sure what this error means? It's the same error on osx and win. I see it's also mentioned in #22378.

```
Setting build arch. This is only useful when pretending to be on another arch, such as for rendering necessary dependencies on a non-native arch. I trust that you know what you're doing.
WARNING:conda_build.config:Setting build arch. This is only useful when pretending to be on another arch, such as for rendering necessary dependencies on a non-native arch. I trust that you know what you're doing.
rendering /Users/runner/work/1/s/recipes/gplately for osx-64
Setting build platform. This is only useful when pretending to be on another platform, such as for rendering necessary dependencies on a non-native platform. I trust that you know what you're doing.
WARNING:conda_build.config:Setting build platform. This is only useful when pretending to be on another platform, such as for rendering necessary dependencies on a non-native platform. I trust that you know what you're doing.
WARNING:/Users/runner/work/1/s/.ci_support/compute_build_graph.py:Dependency platetectonictools, version any is not installable from your channels: ['local', 'conda-forge', 'conda-forge'] with subdir osx-64.  Seeing if we can build it...
rendering /Users/runner/work/1/s/recipes/platetectonictools for osx-64
Traceback (most recent call last):
  File "/Users/runner/work/1/s/.ci_support/build_all.py", line 218, in <module>
    build_all(os.path.join(root_dir, "recipes"), args.arch)
  File "/Users/runner/work/1/s/.ci_support/build_all.py", line 118, in build_all
    build_folders(recipes_dir, folders, arch, channel_urls)
  File "/Users/runner/work/1/s/.ci_support/build_all.py", line 155, in build_folders
    G = construct_graph(recipes_dir, worker=worker, run='build',
  File "/Users/runner/work/1/s/.ci_support/compute_build_graph.py", line 401, in construct_graph
    add_recipe_to_graph(recipe_dir, graph, run, worker, conda_resolve,
  File "/Users/runner/work/1/s/.ci_support/compute_build_graph.py", line 233, in add_recipe_to_graph
    add_dependency_nodes_and_edges(name, graph, run, worker, conda_resolve, config=config,
  File "/Users/runner/work/1/s/.ci_support/compute_build_graph.py", line 466, in add_dependency_nodes_and_edges
    recipe_dir = _buildable(dep, version, recipes_dir, worker, metadata.config,
  File "/Users/runner/work/1/s/.ci_support/compute_build_graph.py", line 442, in _buildable
    available = match_peer_job(ms, m)
  File "/Users/runner/work/1/s/.ci_support/compute_build_graph.py", line 256, in match_peer_job
    if conda_interface.conda_43:
AttributeError: module 'conda_build.conda_interface' has no attribute 'conda_43'
```

**Review by dopplershift — APPROVED** [review]

**Review by dopplershift — APPROVED** [review]

**Comment by dopplershift** [comment]
osx failure looks spurious, no matter since this is noarch.

**Pull request merged**